### ISSUE TITLE
Skipping Button

### ISSUE THREAD

**Issue by vdubedout** [chyrta/AndroidOnboarder#4]
Hi there,

When using the skip button I want to move on something else and not having to slide to the last one and then move to something else.

This can be done changing the onClick

``` java
else if (i == R.id.btn_skip) {
    vpOnboarderPager.setCurrentItem(onboarderAdapter.getCount());
    onSkipButtonPressed();
}
```

to

``` java
else if (i == R.id.btn_skip) {
       onSkipButtonPressed();
}
```

and moving the logic to the abstract method `onSkipButtonPressed()`

``` java
abstract public void onSkipButtonPressed(){
    vpOnboarderPager.setCurrentItem(onboarderAdapter.getCount());
}
```

This way if you want to auto-scroll to last page, call the super.onSkipButtonPressed()
else don't put the super for not scrolling to the last page.

Is it something possible to add in the library or considered out bounds?

**Comment by chyrta** [comment]
@vdubedout Hello, Vincent. Your proposal is essential. I'll plan to work on it.

**Comment by chyrta** [comment]
The issue was resolved in 0.6 version.

**Comment by vdubedout** [comment]
Thanks @chyrta 👍
